modules/gdscript/gdscript.cpp

```diff
@@ -536,6 +536,9 @@ void GDScript::_update_doc() {
 	List<PropertyInfo> props;
 	_get_script_property_list(&props, false);
 	for (int i = 0; i < props.size(); i++) {
+		if (props[i].usage & PROPERTY_USAGE_CATEGORY || props[i].usage & PROPERTY_USAGE_GROUP || props[i].usage & PROPERTY_USAGE_SUBGROUP) {
```

Comment:
```suggestion
		if (props[i].usage & (PROPERTY_USAGE_CATEGORY | PROPERTY_USAGE_GROUP | PROPERTY_USAGE_SUBGROUP)) {
```

99% sure this will work the same, and is arguably more readable, but if anyone disagrees that's fine.

Reply:
@nathanfranke It's not the same thing. Your way will only work if usage has all 3 flags set at once.

Reply:
if the usages are `0b001`, `0b010`, and `0b100`, `x | y | z` will be `0b111` and `usage & (x | y | z)` will be truthy if any flag is enabled.

Reply:
Let's merge as is, but this could be a relevant cleanup PR as it seems we use both styles currently:
```
$ rg "usage & \(PROPERTY_USAGE.*\| PROPERTY_USAGE.*\)" | wc -l
16

$ rg "usage & PROPERTY_USAGE.* \|\| .*usage & PROPERTY_USAGE.*" | wc -l
11
```